core/templates/dev/head/pages/exploration_editor/exploration_save_and_publish_buttons_directive.html

```diff
@@ -32,9 +33,9 @@
   <li ng-if="isEditableOutsideTutorialMode()">
     <div uib-dropdown class="btn-group" style="margin-right: 10px; margin-top: 8px;"
          uib-tooltip="<[getSaveButtonTooltip()]>"
-         tooltip-placement="bottom"
-         ng-class="{'div-disable': !isExplorationSaveable()}"
-         ng-click="showTooltip()">
+         tooltip-placement="right"
+         ng-class="{'disable-save-button': !isExplorationSaveable()}"
+         ng-click="showTooltipForDisabledButtons(disable-save-button)">
```

Comment:
Shouldn't disable-save-button be a string?

Reply:
Fixed.